examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/SamplePaywalls.kt

```diff
@@ -0,0 +1,321 @@
+package com.revenuecat.paywallstester
+
+import com.revenuecat.purchases.Offering
+import com.revenuecat.purchases.Package
+import com.revenuecat.purchases.PackageType
+import com.revenuecat.purchases.models.Period
+import com.revenuecat.purchases.models.Price
+import com.revenuecat.purchases.paywalls.PaywallColor
+import com.revenuecat.purchases.paywalls.PaywallData
+import com.revenuecat.purchases.ui.revenuecatui.data.testdata.TestStoreProduct
+import java.net.URL
+
+class SamplePaywallsLoader {
```

Comment:
Not sure about the structure of this file, so open for suggestions. I don't think it's bad tho